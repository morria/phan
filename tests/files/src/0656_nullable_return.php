<?php
/** @param ?stdClass $x */
function foo656($x) : stdClass {
    return $x;
}
