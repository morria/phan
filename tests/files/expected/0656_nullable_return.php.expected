%s:4 PhanTypeMismatchReturnNullable Returning type ?\stdClass but foo656() is declared to return \stdClass (expected returned value to be non-nullable)
